index.js

```diff
@@ -1,9 +1,11 @@
 const generateId = () => (Math.random() + 1).toString(36).substring(7);
 
-function Book(title, author) {
-  this.title = title;
-  this.author = author;
-  this.id = generateId();
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+    this.id = generateId();
+  }
```

Comment:
- [ ] You should create class methods to add and remove books. This is one of the key requirements of this milestone. Kindly check the [project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m2_plain_js_classes.md#project-requirements)